devops/aws/roles/build-server/tasks/main.yml

```
@@ -5,7 +5,7 @@
   shell: curl https://getsubstrate.io -sSf | bash -s -- --fast
 
 - name: Install subkey
-  shell: cargo install --force subkey --git https://github.com/paritytech/substrate --version 2.0.1 --locked
+  shell: cargo install --force subkey --git https://github.com/paritytech/substrate --version 2.0.2 --locked
```

Should we allow minor version updates with:
```suggestion
  shell: cargo install --force subkey --git https://github.com/paritytech/substrate --version ^2.0.2 --locked
```
So we don't need to deal with this issue in the future?

---

Yeah that is a good idea, I'll confirm that it works and add it.

---

done in [d5a496d](https://github.com/Joystream/joystream/pull/4006/commits/d5a496dd7e38646f158e8aaf10bd796e20d9f6e4)